docs/src/main/sphinx/release/release-381.md

```diff
@@ -0,0 +1,107 @@
+# Release 381 (16 May 2022)
+
+## General
+
+* Add support for fault-tolerant execution with exchange spooling on Azure Blob Storage. ({issue}`12211`)
```

Comment:
@arhimondr @losipiuk @jhlodin  .. this PR introduces new properties and I am pretty sure we need a new section in the docs as well as those properties explained.. also need the same for GCS whenever that is coming.. can you please work on that documentation asap? 

https://trino.io/docs/current/admin/fault-tolerant-execution.html#exchange-manager

Thx